CONTRIBUTING_DOCS.md

```diff
@@ -61,6 +63,8 @@ The submission of a PR kicks off a continuous integration process which includes
 
 However, you can also run this after submitting your PR and experiencing an `htmlproofer` failure from the Semaphore job.
 
+> **Note**: Sometimes `htmlproofer` may deliver errors that do not reproduce. If you get an error from `htmlproofer` that seems incorrect, rerun `htmlproofer`. In addition, sometimes the Semaphore job will return spurious `htmlproofer` errors. In this event, one of the maintainers can kick the job back off again. 
```

Comment:
suggest "...can restart the job." in place of '...can kick the job back off again.'

Reply:
Yes, good point. I think it's better.